<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find next_best_view)/log/rosconsole.config" />
    <node name="set_focus_point" pkg="next_best_view" type="SetFocusPoint.py" output="screen"/>
</launch>
